BDArmory/Modules/MissileFire.cs

```diff
@@ -4171,6 +4171,64 @@ bool SmartPickWeapon_EngagementEnvelope(TargetInfo target)
                             }
                         }
 
+                        if (candidateClass == WeaponClasses.DefenseLaser) //lasers would be a suboptimal choice for strafing attacks, but if nothing else available...
```

Comment:
Didn't this get added already in 1.4.14.0 lines 4121—4172?

Reply:
ok, didn't fully get dev build synced before working. That's quite the spot check fail. Yes, looks like I accidentally duplicated ground lasers